tarodb/assets_store.go

```diff
@@ -1556,6 +1561,13 @@ func (a *AssetStore) QueryParcels(ctx context.Context,
 				return fmt.Errorf("unable to decode tx: %w", err)
 			}
 
+			// Extract the height hint. For backwards compatibility,
```

Comment:
Re this perhaps we should just even store a timestamp/block height here, which can be useful for pagination or doing a sort of historical chain accounting run.